targets/TARGET_RENESAS/TARGET_RZ_A1XX/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if ("RZA1H" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(TARGET_RZ_A1H)
+elseif("RZA1UL" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(TARGET_GR_LYCHEE)
```

it should be 
```suggestion
elseif("GR_LYCHEE" IN_LIST MBED_TARGET_LABELS)
    add_subdirectory(TARGET_GR_LYCHEE)
```
Also keep it in alphabetical order. So:

```
if ("GR_LYCHEE" IN_LIST MBED_TARGET_LABELS)
    add_subdirectory(TARGET_GR_LYCHEE)
elseif("RZ_A1H" IN_LIST MBED_TARGET_LABELS)
    add_subdirectory(TARGET_RZ_A1H)
endif()
```

---

@hugueskamba using 'GR_LYCHEE' causes a compilation error for some reason. I've left the original string as I am not sure what is causing it to break.

---

It has to be that. Paste the compilation error.

---

I must have typed the name wrong because I can't see the error anymore.